### PR TITLE
Update Rule “the-best-chat-tools-for-your-employees/rule”

### PR DESCRIPTION
**Pull request by AttackOnMorty**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by AttackOnMorty** [comment]
Fix indentation so the China box can show correctly.

File: rules/the-best-chat-tools-for-your-employees/use-wechat-for-work.png
Comment (AttackOnMorty) [inline rules/the-best-chat-tools-for-your-employees/use-wechat-for-work.png:1]: - Hide some of my personal chats
- Reveal the chat history with Jim (Checked with @jimmidier, it's okay to do this)

**Review by tiagov8 — APPROVED** [review]
Hey @AttackOnMorty 

Approving this without knowing much about the source of the change. 
1. Please get it checked by someone in China when it's live just to make sure it's all good ;)

**Comment by AttackOnMorty** [comment]
> Hey @AttackOnMorty
> 
> Approving this without knowing much about the source of the change.
> 
> 1. Please get it checked by someone in China when it's live just to make sure it's all good ;)

Done - Checked by @jimmidier

**Pull request merged**